tools/nanopolish/nanopolish_eventalign.xml

```diff
@@ -0,0 +1,111 @@
+<tool id="nanopolish_eventalign" name="Nanopolish eventalign" version="0.1.0">
+    <description>- Align nanopore events to reference k-mers </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input_merged' reads.fasta && 
+        
+        #if $input_reads_raw.extension == 'fast5':
+            mkdir fast5_files && ln -s '$input_reads_raw' fast5_files/read1.fast5 &&
+        #else
+            ln -s '$input_reads_raw' fast5_files.tar.gz &&
+            mkdir fast5_files && tar -xzf fast5_files.tar.gz -C fast5_files &&
+        #end if
+
+        nanopolish index -d fast5_files/ reads.fasta &&
+        ln -s '$input_reads_bam' reads.bam &&
+        ln -s '${input_reads_bam.metadata.bam_index}' reads.bam.bai &&
+        ln -s '$input_genome_fa' genome.fa &&
+        
+        nanopolish eventalign
+        -r reads.fasta
+        -b reads.bam
+        -g genome.fa
+        $verbose
+        $samples
+        $scale_events
+        $SAM
+        $print_read_names
+        #if $window and str($window).strip():
+          -w "${window}"
+        #end if        
+        #if $adv.input_models_fofn:
+          --models-fofn '$input_models_fofn'
+        #end if
+        #if $summary:
+            --summary eventalign-summary.txt
+        #end if
+        > eventalign.out
+
+
+    ]]></command>
+    <inputs>
+      <!-- index inputs -->
+        <param type="data" name="input_merged" format="fasta,fastq" label="Basecalled merged reads.fa"/>
+        <param type="data" name="input_reads_raw" format="h5,fast5.tar.gz" label="Flat archive file of raw fast5 files"/>
+
+        <!-- variants consensus inputs -->
+        <param type="data" name="input_reads_bam" format="bam" label="Reads aligned to the reference genome" help="(-b)" />
+        <param type="data" name="input_genome_fa" format="fasta" label="The reference genome" help="(-g)"/>
+
+        <section name="adv" title="Optional data inputs">
+            <!-- optional inputs -->
+            <param type="data" name="input_models_fofn" format="txt" optional="true" label="Read alternative k-mer models" help="(--models-fofn)" />
+       </section>
+
+        <!-- optional params -->
+        <param name="window" type="text" optional="true" label="find variants in window chromsome_name:start-end" help="(-w)"/>
+
+        <!-- optional flags -->
+        <param name="verbose" type="boolean" truevalue="--verbose" falsevalue="" checked="false" label="verbose output" help="(-v)" />
+        <param name="summary" type="boolean" truevalue="--summary" falsevalue="" checked="true" label="Summarize the alignment of each read/strand" help="(--summary)" />
```

Comment:
this and the following params could be a selection

Reply:
Now switched them all to selection, made them look a bit confusing. If it's ok we stick to boolean.